src/components/PostInfo/PostInfo.jsx

```diff
@@ -2,0 +1,35 @@
+import { UserInfo } from '../UserInfo';
+import { CommentList } from '../CommentList';
+
+import usersFromServer from '../../api/users.json';
+import commentsFromServer from '../../api/comments.json';
+
+import './PostInfo.scss';
+
+export const PostInfo = ({ post }) => {
+  const postComments = commentsFromServer.filter(
+    comment => comment.postId === post.id,
+  );
+
+  return (
+    <div className="PostInfo">
+      <div className="PostInfo__header">
+        <h3 className="PostInfo__title">{post.title}</h3>
+
+        <UserInfo
+          user={usersFromServer.find(user => user.id === post.userId)}
```

Comment:
Ensure that `usersFromServer` is an array of user objects with an `id` field to correctly find the user associated with the post.